src/Persistence/Models/DataType.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Models;
+
+#pragma warning disable CA1720 // Identifier contains type name
+public enum DataType
```

Comment:
Can this get out of sync with server side?

Reply:
yes, I was thinking the same. We could use `string` instead, what do you think

Reply:
string is safer I think